lib/gyro/liquidgen/generator.rb

```diff
@@ -89,41 +100,21 @@ def generate_enum(enum_name, output)
       end
 
       def render_entity(context)
-        # Parse object template
-        entity_template_path = @template_dir + 'root.liquid'
-        Gyro::Error.exit_with_error('Bad template directory content ! Your template need to include root.liquid file') unless entity_template_path.exist?
-        entity_template_string = entity_template_path.read
-        entity_template = Liquid::Template.parse(entity_template_string)
-        entity_template.render(context, filters: [CustomFilters])
+        @entity_template.render(context, filters: [CustomFilters])
                                 .gsub(/^ +$/, '')
       end
 
       def render_filename(context)
-        # Parse object template name
-        filename_template_path = (@template_dir + 'filename.liquid')
-        Gyro::Error.exit_with_error('Bad template directory content ! Your template need to include filename.liquid file') unless filename_template_path.exist?
-        filename_template_string = filename_template_path.readlines.first
-        filename_template = Liquid::Template.parse(filename_template_string)
-        filename_template.render(context).chomp
+        @entity_filename_template.render(context).chomp
```

Comment:
Or better, emit a warning (or even error out) to tell the user to fix the `entity_filename.liquid` template, because after all that's the template author's responsibility 😉